packages/grant-explorer/src/features/discovery/RoundCard.tsx

```diff
@@ -22,86 +23,143 @@ type RoundCardProps = {
   round: RoundOverview;
 };
 
-const RoundCard = (props: RoundCardProps) => {
-  const daysLeft = getDaysLeft(Number(props.round.roundEndTime));
-  const chainIdEnumValue = ChainId[props.round.chainId as keyof typeof ChainId];
+const RoundCard = ({ round }: RoundCardProps) => {
+  const {
+    id,
+    chainId,
+    matchAmount,
+    projects,
+    payoutStrategy,
+    roundMetadata,
+    roundEndTime,
+    applicationsEndTime,
+    token,
+  } = round;
+  const daysLeft = getDaysLeft(Number(roundEndTime));
+  const daysLeftToApply = Math.round(getDaysLeft(Number(applicationsEndTime)));
 
+  // Can we simplify this? Would `days < 1000` do the same thing?
+  const isValidRoundEndTime = !isInfiniteDate(
+    new Date(parseInt(roundEndTime, 10) * 1000)
+  );
+
+  const chainIdEnumValue = ChainId[chainId as keyof typeof ChainId];
   const { data } = useToken({
-    address: getAddress(props.round.token),
-    chainId: Number(props.round.chainId),
+    address: getAddress(token),
+    chainId: Number(chainId),
   });
 
   const nativePayoutToken = votingTokens.find(
-    (t) =>
-      t.chainId === chainIdEnumValue &&
-      t.address === getAddress(props.round.token)
+    (t) => t.chainId === chainIdEnumValue && t.address === getAddress(token)
   );
 
   const tokenData = data ?? {
     ...nativePayoutToken,
     symbol: nativePayoutToken?.name ?? "ETH",
   };
 
-  const approvedApplicationsCount = props.round.projects?.length ?? 0;
-
+  const approvedApplicationsCount = projects?.length ?? 0;
   return (
     <BasicCard className="w-full">
       <a
         target="_blank"
-        href={`/#/round/${chainIdEnumValue}/${props.round.id}`}
+        href={`/#/round/${chainIdEnumValue}/${id}`}
         data-testid="round-card"
       >
-        <CardHeader>
-          <RoundBanner roundId={props.round.id} />
+        <CardHeader className="relative">
+          <RoundBanner roundId={id} />
+          {daysLeftToApply > 0 ? (
+            <Badge
+              color="green"
+              rounded="full"
+              className="absolute top-3 right-3"
+            >
+              Apply!
+            </Badge>
+          ) : null}
```

Comment:
`condition ? <element/> : null` can be simplified as `condition && <element/>`

```suggestion
          {daysLeftToApply > 0 && 
            <Badge
              color="green"
              rounded="full"
              className="absolute top-3 right-3"
            >
              Apply!
            </Badge>}
```